src/Nextjs.ts

```diff
@@ -134,6 +140,7 @@ export class Nextjs extends Construct {
       tempBuildDir,
       nextBuild: this.nextBuild,
       lambda: props.defaults?.lambda,
+      cacheBucket: this.staticAssetBucket,
```

Comment:
we should create a separate bucket here, maybe `CacheBucket`. See diagram [here](https://github.com/serverless-stack/open-next/blob/main/README.md#recommended-infrastructure-on-aws) how there are 2 different buckets.

Reply:
I'll create a new bucket. I'm slightly struggling with how much of `NextjsAssetsDeployment` to refactor so I can reuse it for this `CacheBucket`. 

> ...Create an S3 bucket and **upload the content in the .open-next/cache folder to the root of the bucket**...

I'm leaning towards slightly refactoring NextjsAssetsDeployment so that it can be re-used, because I'm assuming (given [this open-next documentation says that it's OK to share buckets](https://github.com/serverless-stack/open-next/blob/main/README.md#reusing-same-bucket-for-asset-and-cache)) all other settings could be shared (e.g. cache configuration). I actually don't have a great sense here, perhaps adding caching configuration to this bucket is confusing because it's not needed. 

Obviously open to direction / advice here!

Reply:
Added a separate bucket. Deployment is much simpler than the static asset bucket (there is a "rewriter" for that, I'm not sure if that's something we need for the cache bucket as well).

Reply:
I don't believe so, at least the docs don't mention it.